src/LanguageServer.Engine/Documents/Workspace.cs

```diff
@@ -172,13 +172,11 @@ public async Task<ProjectDocument> GetProjectDocument(Uri documentUri, bool relo
                 isNewProject = true;
```

Comment:
Is this flag always set to `true`?

Reply:
From the look of it, only if the factory delegate runs?

Reply:
Uh, I see, my bad)